content/docs/contributing/to-code/git-workflow.md

```diff
@@ -91,7 +91,16 @@ At a high level, the workflow is:
 
 Now, you're ready to do some work! After you've made some changes and saved your work, it's time to submit it as a pull request (PR) for review.
 
-1. First, stage (add) the new or modified code files, commit your changes, and submit a pull request to your fork (origin) with the following commands. (**Check which branch you are on with `git branch` first!**):
+1. (Optional): First, depending on how much time has passed since your branch was initially created, you might want to merge the latest from `upstream/development` into your branch. This will ensure your changes don't conflict with any other recent code commits, and also gives your automated review (AR) the best chance of success the closer your branch is to the latest development snapshot.
+
+    You can do this with the following commands (**Check which branch you are on with `git branch` first!**):
+
+    ```bash
+    git fetch upstream --all
+    git pull
+    ```
```

Comment:
don't you need "git merge development" in here as well?

Reply:
I've always just done `git pull` and it automatically merges in for me based on the branch I created my branch off of. Is that a local setting that I enabled a long time ago and just forgot about?

Reply:
could be, sounds like you have a different workflow to me - I do the merge manually

Reply:
Yeah, I'll add the manual merge step to be safe. If it's already merged by the pull it would just be a no-op.

Reply:
yeah, good point, sounds good

Reply:
might be worth mentioning DCO signoff as well

Reply:
Oh good point, they mention it in the next step, but should I add the `--signoff` to the end of the merge command as well?

Reply:
I think that will work, but I'm not 100% sure, because I do

git merge --no-commit development
git commit -s -m "merge from development"

Reply:
I do the `--signoff` when I'm doing integrations, so it should be good. And then the next step does:

```
git commit -s -m "<commmit_message>"
```